docs/overview.md

```diff
@@ -3,60 +3,36 @@ id: overview
-* [**Fast completions**](#fast-completions): completions suggestions should
-  respond in ~200ms, even at the cost of occasionally wrong results.
-* [**Low CPU and memory usage**](#low-cpu-and-memory-usage): indexing should run
+- [**Low CPU and memory usage**](#low-cpu-and-memory-usage): indexing should run
   in the background and not get in your way from coding.
-* [**Solid refactoring**](#solid-refactoring): rename, organize imports and
-  insert type annotation should always leave the codebase in a compiling state.
+
+We acknowledge that simple installation and robust navigation alone is not
+enough to make a great language server. A great language server should also
+support correct diagnostics, fast completions and solid refactorings. However,
+we believe navigation is an important first milestone that needs full attention
+before working on other features.
 
 ## Simple installation
 
 The first you do with an IDE is boring but critical: import a project. The
 Language Server Protocol does not provide utilities to extract metadata from a
-build tool (module source files, dependencies, configuration data), so we are
-still exploring what is the best option.
-
-* [x] sbt-metals: plugin that provides a `metalsSetup` task to export sbt
-    build metadata.
-
-* [ ] [BSP][]: LSP-inspired protocol to standardize on communication between a
-  language server and build tool. In theory, should enable automatic
-  importing of projects without custom build plugins or manual installation
-  steps.
-
-## Correct diagnostics
-
-Red squiggles are one of the most basic but also difficult-to-support features
-in an IDE. The Scala compiler was not designed for interactivity, which makes
-fast and correct diagnostics difficult to support in the editor. We are still
-unsure what the best model is for diagnostics, so we are still exploring what is
-the best option.
-
-* [x] Scala Presentation Compiler: interactively publish squiggles as you type.
-      Provides low latency but may get off-sync with build diagnostics.
-  * Disabled by default. Can be enabled with configuration option
-    `scalac.diagnostics.enabled=true`.
-  * Requires [`metalsSetup`](#metalssetup).
-* [x] sbt server: sbt 1.1 added LSP support so that build errors are reported
-      directly to the editor. Metals can connect to sbt server as a client to
-      trigger batch compilation on file save. Guarantees diagnostics are always
-      in-sync with build diagnostics but at the cost of higher latency.
-  * Disabled by default. Can be enabled with configuration option
-    `sbt.enabled=true`.
-  * Requires a running sbt 1.1 server or later.
-* [x] Linting with [Scalafix][].
-  * Requires [semanticdb-scalac](#semanticdb-scalac).
-  * Supports both built-in and custom linter rules.
+build tool (source directories, dependencies, compiler flags) so we are still
+exploring what is the best option.
+
+- [x] sbt-metals: plugin that provides a `metalsSetup` task to export sbt build
```

Comment:
I think having these checkboxes isn't helpful. Pretty much everything is under exploration so they give a false sense of accomplishment.

Reply:
Good point, I will remove them 👍